src/test/resources/capabilities-parallel-browsers.yml

```diff
@@ -0,0 +1,48 @@
+testEndpoint: https://bstackdemo.com
+
+driverType: cloudDriver
+
+cloudDriver:
+  hubUrl: https://hub-cloud.browserstack.com/wd/hub
+  user: BROWSERSTACK_USERNAME
+  accessKey: BROWSERSTACK_ACCESSKEY
+  common_capabilities:
+    project: BrowserStack Demo Repository
+    buildPrefix: browserstack-examples-testng
```

Comment:
change to **browserstack-examples-cucumber-testng**

Reply:
Corrected the Build Prefix name to browserstack-examples-cucumber-testng